website/src/components/InlineSVG.astro

```diff
@@ -1,18 +1,19 @@
 ---
 export interface Props {
-	src: string;
+  src: string;
 }
 
 const { src } = Astro.props as Props;
 
 const files = await import.meta.glob<string>("/src/assets/svg/**/*.svg", {
-	as: "raw",
```

Comment:
`as: "raw"` was deprecated in Vite, so I've replaced it for the recommended `query: "?raw"`